src/core/org/apache/jmeter/threads/JMeterThread.java

```diff
@@ -123,8 +123,10 @@
 
     private long endTime = 0;
 
-    private boolean scheduler = false;
+    private boolean isSameUserOnNextIteration = false;
```

Comment:
Is this `final` or not?
If the variable is not final, when what sense does it make to add a constructor argument for it?